cmd/admin-handlers_test.go

```diff
@@ -162,3 +163,151 @@ func TestServiceStopHandler(t *testing.T) {
 func TestServiceRestartHandler(t *testing.T) {
 	testServicesCmdHandler(restartCmd, t)
 }
+
```

Comment:
This file lacks comments. Every function (public and private )requires comment. Even code blocks like for and if.